pkg/templates/templates.go

```diff
@@ -34,19 +35,23 @@ const (
 type EncryptionMode uint8
 
 const (
-	// Disables the "protect" method.
+	// Disables the "protect" template function.
```

Comment:
Optional Comment:  If I were doing this, I would have considered having two separate config params bool encryption_enabled and bool decryption_enabled ,  instead of overloading one param `encryptionmode`.

cleaner and could allow for a possible  future case where a controller may need to support both modes,

Reply:
Good idea! I'll make the change.